CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,63 @@
+# Google Open Source Community Guidelines
```

Comment:
I think the code of conduct should use this text:
https://opensource.google/docs/releasing/template/CODE_OF_CONDUCT/

Reply:
Thanks for catching this @ronaldfw .  Should be better now...  :-)